### PR TITLE
Add onchange autocomplete selected value JS event

### PR DESCRIPTION
**Pull request by davidromani**
This is useful when you need to do something more async

### CODE REVIEW

File: Resources/doc/reference/form_types.rst [hunk 1]
```diff
@@ -317,6 +317,10 @@ template
     {# change the default selection format #}
     {% block sonata_type_model_autocomplete_selection_format %}'<b>'+item.label+'</b>'{% endblock %}
 
+.. tip::
+
+    A jQuery event is fired after when autocomplete selected value has been changed (``sonata.autocomplete_new_element_selected``).
```
Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: ``after when``

**Comment by OskarStark** [comment]
i like your PR :+1:

**Comment by davidromani** [comment]
@OskarStark thx, but I'm still in trouble because I don't know how to catch this event on browser-side

**Comment by OskarStark** [comment]
ping @rande

**Comment by core23** [comment]
Can you rename the event to **sonata.autocomplete_element_selected**? @davidromani

**Comment by core23** [comment]
Ping @davidromani

**Comment by davidromani** [comment]
Sorry, I'm quite busy right now, can you do it please? @core23

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**